README.md

```diff
@@ -1,25 +1,35 @@
-# TASK1
+# TASK2
 
 ## Build image 
 
 ```
-docker build -t [IMAGE_NAME] .
+docker build -t myimage .
 ```
 
-## Run container
+## Run docker compose
 
 ```
-docker run -p 8080:8080 -e KEYWORD_ONE=[FIRST_VARIABLE] -e KEYWORD_TWO=[SECOND_VARIABLE] [IMAGE_NAME]
+docker-compose run --service-ports -e POSTGRES_PASSWORD=[Your postgres pass] -e POSTGRES_USER=[Your postgres user] web
 ```
-
-## Expected result
-Go to localhost:8080.
-You must see
+There is default name of your database :"postgres"
+####Go to localhost:8080/actuator/health
```

Comment:
@blackavgn Please check formatting for ####

Reply:
@blackavgn what 'Go to localhost' mean? I suppose that 'Go' mean 'open in browser'? Let's clarify it. Better - tell which HTTP method we should use